source/reference/minio-server/settings/additional-settings.rst

```diff
@@ -0,0 +1,154 @@
+.. _minio-server-envvar-additional-settings:
+
+===================
+Additional Settings
```

Comment:
TBH I'm not sure that there is anything here that wouldn't also possibly belong in Core Settings. We talk about ILM workers there, but batch workers here. Compression is kind of a core feature.